tests/rest/client/v2_alpha/test_register.py

```diff
@@ -405,6 +406,56 @@ def test_manual_expire(self):
             channel.json_body["errcode"], Codes.EXPIRED_ACCOUNT, channel.result
         )
 
+    def test_logging_out_expired_user(self):
```

Comment:
I think there should also be a test for access token expiration since this PR is doing stuff around that.